spec/01-unit/09-balancer/04-round_robin_spec.lua

```diff
@@ -1039,6 +1040,7 @@ describe("[round robin balancer]", function()
       -- expire the existing record
       record.expire = 0
       record.expired = true
+      sleep(0.2)
```

Comment:
Should add a comment to explain why it is 0.2 second.

Reply:
got it, will add it later, here it means wait the record to become expired after 0.2s

Reply:
done